docs/input/docs/reference/configuration.md

```diff
@@ -53,6 +53,8 @@ commit-date-format: yyyy-MM-dd
 merge-message-formats: {}
 update-build-number: true
 semantic-version-format: Strict
+strategies:
+- ConfigNext, MergeMessage, TaggedCommit, TrackReleaseBranches, VersionInBranchName
```

Comment:
Arrays in YAML are either wrapped in `[]`:

```suggestion
strategies: [ConfigNext, MergeMessage, TaggedCommit, TrackReleaseBranches, VersionInBranchName]
```

…or on separate lines starting with `-`:

```suggestion
strategies:
  - ConfigNext
  - MergeMessage
  - TaggedCommit
  - TrackReleaseBranches
  - VersionInBranchName
```

Reply:
done